ucm2/sof-hda-dsp/HiFi.conf

```diff
@@ -1,22 +1,33 @@
 # Use case Configuration for sof-hda-dsp
 
-SectionVerb {
-	EnableSequence [
-		cset "name='Master Playback Switch' on"
-		cset "name='Capture Switch' on"
-	]
-}
-
 SectionDevice."Headphone" {
 	Comment "Headphone"
 
-	EnableSequence [
-		cset "name='Headphone Playback Switch' on"
-	]
+	If.1 {
+		Condition {
+			Type ControlExists
+			Control "name='Input Source'"
+		}
+		True {
+			EnableSequence [
+				cset "name='Headphone Playback Switch' on"
+			]
 
-	DisableSequence [
-		cset "name='Headphone Playback Switch' off"
-	]
+			DisableSequence [
+				cset "name='Headphone Playback Switch' off"
+				cset "name='Input Source' 1"
```

Comment:
We can use the enum name here. The value '1' is not self-explaining. Use _cset "name='Input Source' Mic"_ or so...

Reply:
ok